Sources/EventSource/EventParser.swift

```diff
@@ -9,12 +9,13 @@
 import Foundation
 
 public protocol EventParser: Sendable {
-    func parse(_ data: Data) -> [EVEvent]
+    func parse(_ data: Data) async -> [EVEvent]
```

Comment:
Should this function be marked as async?

Reply:
I allowed async function in `EventParser` protocol to allow conformance for an actor that run asynchronous